BlogTemplate/Pages/Edit.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@page "{slug}"
+@using BlogTemplate
+@using BlogTemplate.Models
+@inject Blog Blog
+@model EditModel
+@using Microsoft.AspNetCore.Mvc.RazorPages
+@addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
+
+@{
+    ViewData["Title"] = "Edit Post";
+}
+
+<article>
+    <h2>Edit Post</h2>
+    <form method="post">
+        <p><h3><label for="Title">Title</label></h3></p>
```

Comment:
&lt;h3> doesn't really belong in a &lt;p>.  Neither do the &lt;input> fields below.